src/sentry/api/endpoints/organization_details.py

```diff
@@ -416,7 +417,9 @@ def save(self):
                     if flag_has_changed(org, f):
                         changed_data[f] = f"to {v}"
 
-        org.save()
+        with outbox_context(transaction.atomic(), flush=False):
+            org.save()
```

Comment:
Hmm, I'm not sure I like this.  Is there a reason this isn't either
1.  outboxes created in save
or
2.  In a domain function?

I worry a lot about having outbox creation in controllers.

Reply:
Fair enough, I'll move this to use a domain function. There's one other change like this in this PR when we restore an org from deletion that I'll need to pull in as well.